src/gui/editors/AutomationEditor.cpp

```diff
@@ -1055,58 +1058,55 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 
 	p.fillRect(0, TOP_MARGIN, VALUES_WIDTH, height() - TOP_MARGIN, m_scaleColor);
 
+	if (!validClip())
+	{
+		const auto icon = embed::getIconPixmap("pr_no_clip");
+		const int x = (width() - icon.width()) / 2;
+		const int y = (height() - icon.height()) / 2;
+		p.drawPixmap(x, y, icon);
+
+		p.setPen(QApplication::palette().color(QPalette::Active, QPalette::Text));
+		QRect textRect(0, y + icon.height() + 5, width(), 30);
+		p.drawText(textRect, Qt::AlignHCenter | Qt::AlignTop,
+			tr("Double-click on an automation clip in Song Editor to open it here"));
+		return;
+	}
+
 	// print value numbers
 	int font_height = p.fontMetrics().height();
 	auto text_flags = (Qt::Alignment)(Qt::AlignRight | Qt::AlignVCenter);
 
-	if( validClip() )
+	if (m_y_auto)
```

Comment:
I wouldn't touch this part of the code, just to keep the commit to the point. But that's a personal opinion.

Reply:
Ok other devs think this is better, so I withdraw my opinion